README.md

```diff
@@ -47,110 +72,64 @@ The Lua streaming UDF will use the argument you pass to it in its calculations b
   },
 }
 ```
+The `key` is a hash used to group the results for reduction. The value is a map of the returned fields. In the map, the key is the alias of the field.
 
-The client does not calculate average values, but that can be accomplished as the last step on the client.
+Regardless of the aggregations used, a count of records in the group will always be returned in `count`. Avoid using this name in your requests.
```

Comment:
This is not the case since the latest changes. If one needs an absolute count, they should define a field like:
```js
"total_count": {"func": "count", "expr": "1"}
```
This was done for both performance and consistence reasons.